source/aod.rst

```diff
@@ -0,0 +1,366 @@
+
+  Access via the EC3 portal `EC3 portal`_. 
+
+* `Kubernetes <https://kubernetes.io/>`_ is an open-source container-orchestration
+  system for automating application deployment, scaling, and management.
+  Through this framework users can create a containerized version of workflows
+  and services that can be deployed in the EGI Infrastructure.  
+
+  Access via the EC3 portal `EC3 portal`_. 
+
+* The `AppDB VMOps dashboard <https://dashboard.appdb.egi.eu/>`_, a framework to
+  perform Virtual Machine (VM) management operations on the EGI Federated Cloud.
+  The graphical interfaces of the dashboard allows the user to choose VMs from
+  the AppDB Catalogue, to define an execution topology for them, and then to
+  instantiate them on infrastructure as a service clouds. 
+
+  The EGI VMOps dashboard is available [https://dashboard.appdb.egi.eu/ here]
+
+AoD portals/gateways
+--------------------
+
+Applications of the service are accessed via three different portals/gateways:
+
+FutureGateway Science Gateway
+:::::::::::::::::::::::::::::
+
+The `FutureGateway Science Gateway (FGSG) <http://www.catania-science-gateways.it/>`_
+is a programmable interface of a RESTful API Server, compliant with CSGF APIs
+specifications, able to provide an easy acces to the PaaS layer by leveraging
+on recent Web technologies. 
+
+* The FGSG incorporates several applications and offers these "as services"
+  for the user. 
+* The FGSG provides users with an intuitive web interface to execute dockerized
+  applications on the cloud resources of the platform. 
+
+  * The FGSG take cares of starting the job in one of the available resources
+    of the platform, of transferring the needed files (e.g. executable, input
+    files, etc.) and downloading the output(s) in behalf of the users
+    
+* FGSG is open source and released under the Apache 2.0 license.
+
+Access `the FGSG integrated in the AoD <https://fgsg.ct.infn.it/egissod/web/ssod>`_.
+
+WS-PGRADE
+:::::::::
+
+The `WS-PGRADE <http://guse.hu/about/architecture/ws-pgrade>`_ (Web Services
+Parallel Grid Runtime and Developer Environment Portal) is the Liferay-based
+web portal (WS-PGRADE web application) of gUSE, wich also includes a graphical
+portal service. 
+
+* WS-PGRADE is a Web portal hosted in a standard portal framework, using the
+  client APIs of gUSE services to turn user requests into sequences of gUSE
+  specific Web service calls. 
+* WS-PGRADE is integrated with the cloud resources of the platform, and
+  provides a 'job wizard' interface for the user. 
+* Through the wizard one can define with a few clicks a computational job that
+  WS-PGRADE will execute on the cloud resources. 
+
+  * The environment takes care of instantiation of Virtual Machine image for
+    the job, for sending input data for the job, and for the retrieval of job
+    outputs.
+
+Access `the WS-PGRADE/gUSE integrated in the AoD <https://guse.egi.eu/c/portal/login?openIdLogin=true>`_.
```

Comment:
The https://guse.egi.eu/c/portal/login page is broken as the cert is for ltos-gateway.lpds.sztaki.hu.
Also I fear that this Liferay instance may be quite old and full of vulnerabilities, is it updated/maintained?

Reply:
Probably not well maintained, we were discussing whether this can be removed completely, I think we should.